src/axolotl/prompt_tokenizers.py

```diff
@@ -85,7 +85,11 @@ def _tokenize(self, prompt: str, add_eos_token=True, strip_bos_token=False):
             result["input_ids"].append(self.tokenizer.eos_token_id)
             result["attention_mask"].append(1)
 
-        if result["input_ids"][0] == self.tokenizer.bos_token_id and strip_bos_token:
+        if (
+            len(result["input_ids"]) > 0
```

Comment:
Could we also add a LOG.info / warn if length is 0? Same like a previous PR?

Reply:
we already warn about in line 78